anomalib/models/patchcore/torch_model.py

```diff
@@ -157,6 +157,8 @@ def nearest_neighbors(self, embedding: Tensor, n_neighbors: int) -> Tuple[Tensor
         if n_neighbors == 1:
             # when n_neighbors is 1, speed up computation by using min instead of topk
             patch_scores, locations = distances.min(1)
+            # compute anomaly scoree expects patch_scores and locations to be a 2D tensor
+            patch_scores, locations = patch_scores.unsqueeze(1), locations.unsqueeze(1)
```

Comment:
I don't understand why this is necessary. The tensor gets reshaped in the forward pass [here](https://github.com/openvinotoolkit/anomalib/blob/hotfix/main/anomalib/models/patchcore/torch_model.py#L83), right after they're returned from this method, so what's the point of adding the dimension?

Reply:
After fixing L189, this line is not needed anymore. You're right